### PR TITLE
OCPBUGS-12622-br-ex: Added note to creating-manifest-file-customized-…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16+

Issue:
[OSDOCS-12622](https://issues.redhat.com/browse/OSDOCS-12622)

Link to docs preview:

* [Setting up the environment for an OpenShift installation](https://84665--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/ipi/ipi-install-installation-workflow.html#creating-manifest-file-customized-br-ex-bridge_ipi-install-installation-workflow)
* [**Postinstallation configuration**](https://84665--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.html#creating-manifest-file-customized-br-ex-bridge_ipi-install-post-installation-configuration)
* [Installing a user-provisioned cluster on bare metal](https://84665--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/upi/installing-bare-metal.html#creating-manifest-file-customized-br-ex-bridge_installing-bare-metal)
* [Installing a user-provisioned bare metal cluster with network customizations](https://84665--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.html#creating-manifest-file-customized-br-ex-bridge_installing-bare-metal-network-customizations)
* [Installing a user-provisioned bare metal cluster on a restricted network](https://84665--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.html#creating-manifest-file-customized-br-ex-bridge_installing-restricted-networks-bare-metal)

- [x] SME has approved this change.
- [x] QE has approved this change.

### CODE REVIEW

**Review by cybertron — COMMENTED** [review]
Was this related to my comment about not changing the IP on day 2? If so, I was looking for a note further down here (around line 156) that just says to ensure the primary node IP doesn't get changed. Static or DHCP are kind of irrelevant because in either case you need to make sure the address doesn't change.

**Review by cybertron — COMMENTED** [review]
/lgtm

**Comment by rbbratta** [comment]
This is [OSDOCS-12622](https://issues.redhat.com/browse/OSDOCS-12622) not the OCPBUGS same number?

Can we retitle in case we confuse the bots?

**Comment by dfitzmau** [comment]
> This is [OSDOCS-12622](https://issues.redhat.com/browse/OSDOCS-12622) not the OCPBUGS same number?
> 
> Can we retitle in case we confuse the bots?

Yes, this is the same Jira. I can create a separate subtask if that would work better but it would mean I need to raise a new PR.

File: modules/creating-manifest-file-customized-br-ex-bridge.adoc [hunk 1]
```diff
@@ -151,6 +151,11 @@ endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
 ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
 . Create a `NodeNetworkConfigurationPolicy` (NNCP) CR and define a customized `br-ex` bridge network configuration. Depending on your needs, ensure that you set a masquerade IP for either the `ipv4.address.ip`, `ipv6.address.ip`, or both parameters. A masquerade IP address must match an in-use IP address block.
 +
+[IMPORTANT]
+====
+As a post-installation task, you can configure most parameters for a customized `br-ex` bridge that you defined in an existing NNCP CR except for the IP address.
```
Comment (skrthomas) [inline modules/creating-manifest-file-customized-br-ex-bridge.adoc:156]:
```suggestion
As a post-installation task, you can configure most parameters for a customized `br-ex` bridge that you defined in an existing NNCP CR, except for the IP address.
```

**Review by skrthomas — COMMENTED** [review]
Lil nit. Looks good to me otherwise :)

**Pull request merged**